dair_pll/inertia.py

```diff
@@ -237,22 +390,41 @@ def drake_to_pi(spatial_inertia: DrakeSpatialInertia) -> Tensor:
             number_to_float(I_BBcm_B[index[0], index[1]])
             for index in INERTIA_INDICES
         ]
-        pi = Tensor([mass] + mass_list + inertia_list)
-        return pi
+        pi_cm = Tensor([mass] + mass_list + inertia_list)
+        return pi_cm
+
+    @staticmethod
+    def drake_to_pi_o(spatial_inertia: DrakeSpatialInertia) -> Tensor:
```

Comment:
is there a reason this isn't implemented as a passthrough like `pi_cm_to_theta`?

Reply:
You're correct in that the current implementation relying on Drake's `CalcRotationalInertia` and `Shift` (for `drake_to_pi_cm`) is equivalent to a pass-through, e.g. `drake_to_pi_o(*) = pi_cm_to_pi_o(drake_to_pi_cm(*))`.  Since either is technically correct, I chose to rely on Drake's infrastructure rather than ours just since theirs is maintained by a team and might be less error-prone and more efficient.  However I'm happy to switch to a pass-through approach if you have reasons to prefer it, even if just stylistic.

Reply:
The negative thing about the current implementation is that it's a copy-paste implementation, i.e. literally implemented by copying and pasting working code. Copy/paste tends to be bad because if we ever discover a bug with or want to change both functions, there's a risk of forgetting to do the same to the other function and there's also a doubled risk of making a typo in either one each time a modification is made; they also typically increase the total line-count of the file, such that there is more code to maintain.

A third option that minimizes copy/paste _and_ uses `CalcRotationalInertia ` would be to implement a helper function `drake_inertial_components_to_pi(mass, p_BoBcm_B, I_BoBi_B)` that contains the code from `mass_list = ...` onward, and reimplement both `drake_to_pi_o` and `drake_to_pi_o` as 2-liners that call this function.

Reply:
I like that suggestion.  I'll do that now, thanks.